backend/dining/api_wrapper.py

```diff
@@ -57,57 +57,29 @@ def request(self, *args, **kwargs):
             raise APIError("Dining: Connection timeout")
 
     def get_venues(self):
-        results = []
-        venues_route = OPEN_DATA_ENDPOINTS["VENUES"]
-        response = self.request("GET", venues_route)
-        if response.status_code != 200:
-            raise APIError("Dining: Error connecting to API")
-        venues = response.json()["result_data"]["campuses"]["203"]["cafes"]
-        for key, value in venues.items():
-            # Cleaning up json response
-            venue = Venue.objects.filter(venue_id=key).first()
-            value["name"] = venue.name
-            value["image"] = venue.image_url if venue else None
-
-            value["id"] = int(key)
-            remove_items = [
-                "cor_icons",
-                "city",
-                "state",
-                "zip",
-                "latitude",
-                "longitude",
-                "description",
-                "message",
-                "eod",
-                "timezone",
-                "menu_type",
-                "menu_html",
-                "location_detail",
-                "weekly_schedule",
-            ]
-            [value.pop(item) for item in remove_items]
-            for day in value["days"]:
-                day.pop("message")
-                removed_dayparts = set()
-                for i in range(len(day["dayparts"])):
-                    daypart = day["dayparts"][i]
-                    [daypart.pop(item) for item in ["id", "hide"]]
-                    if not daypart["starttime"]:
-                        removed_dayparts.add(i)
-                        continue
-                    for time in ["starttime", "endtime"]:
-                        daypart[time] = datetime.datetime.strptime(
-                            day["date"] + "T" + daypart[time], "%Y-%m-%dT%H:%M"
-                        )
-                # Remove empty dayparts (unavailable meal times)
-                day["dayparts"] = [
-                    day["dayparts"][i]
-                    for i in range(len(day["dayparts"]))
-                    if i not in removed_dayparts
+        venues = Venue.objects.all()
+        res = [None] * len(venues)
+        for i, venue in enumerate(venues):
+            venues_obj = {"id": venue.venue_id, "name": venue.name, "image": venue.image_url}
+
+            days = []
+            # get info for a week
+            for day_offset in range(7):
+                days_obj = dict()
+
+                date = datetime.datetime.today() + datetime.timedelta(days=day_offset)
+                date = date.strftime("%Y-%m-%d")
+                days_obj["date"] = date
+
+                # get starttimes and endtimes from DiningMenu objects
+                days_obj["dayparts"] = [
+                    {"starttime": menu.start_time, "endtime": menu.end_time, "label": menu.service}
```

Comment:
Do we want to change the object keys to match our variables' names and have iOS change it on their side?

Reply:
For now, let's keep the variable names as is, because changes to iOS and Android actually take a while to propagate. Later we should make this more Django native but now it'll be fine haha